README.md

```diff
@@ -150,6 +150,117 @@ rescue
 end
 ```
 
+## Insights Automatic Instrumentation
+
+Honeybadger Insights allows you to automatically track various events in your
+application. To enable Insights automatic instrumentation, add the following to
+your configuration:
+
+```elixir
+config :honeybadger,
+  insights_enabled: true
+```
+
+### Supported Libraries
+
+Honeybadger automatically instruments the following libraries when they are available:
+
+* Absinthe: GraphQL query execution
+* Ecto: Database queries
+* Finch: HTTP client requests
```

Comment:
Few apps use Finch directly, it's usually through a wrapper like Req. That may be helpful to mention here when people are scanning for supported packages.

Reply:
Thanks for making me aware of that. I updated the comment. I also looked into who else uses Finch, and it looks like Tesla has an adapter. I added some code to not send the Tesla insights event if [Finch is used as the adapter](https://github.com/honeybadger-io/honeybadger-elixir/pull/591/commits/0dfe1f27605eceba12f83e878a6d4690ad51bd73).

Reply:
It may cause errors depending on the load order, especially for dynamically loaded modules in development. Declaring the list of atoms is a safe work around.